docker/build/edxapp/docker-run.sh

```diff
@@ -183,6 +183,15 @@ run_command () {
       eval "${edxpython} ${manage} ${service} --settings=bdu add_dark_lang_config --langs ${DARK_LANGS:-en,fr,zh-cn,es-419,uk,ru,pt-br,it-it}"
     ;;
 
+    lms-sync-mailchimp-announcement-list)
+      echo "Adding mailchimp user list id"
+      wait_for_mysql $service
+      source /edx/app/edxapp/edxapp_env
+      local edxpython="/edx/bin/python.edxapp"
+      local manage="/edx/bin/manage.edxapp"
+      eval "${edxpython} ${manage} ${service} --settings=bdu mailchimp_sync_announcement --langs ${BDU_PORTAL_MAILCHIMP_USER_LIST_ID} ${BDU_PORTAL_MAILCHIMP_API_KEY}"
```

Comment:
```suggestion
      eval "${edxpython} ${manage} ${service} --settings=bdu mailchimp_sync_announcement --list ${BDU_PORTAL_MAILCHIMP_USER_LIST_ID} --key ${BDU_PORTAL_MAILCHIMP_API_KEY}"
```

https://github.com/edx/edx-platform/blob/open-release/juniper.master/lms/djangoapps/mailing/management/commands/mailchimp_sync_announcements.py#L23-L29